### PR TITLE
!joinme feacture

### PR DESCRIPTION
**Pull request by azlux**
code for the issue/feacture #142

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Command | Description | Arguments | Admin | Example
 **help** | Displays this list of commands in Mumble chat. | None | No | `!help`
 **volume** | Either outputs the current volume or changes the current volume. If desired volume is not provided, the current volume will be displayed in chat. Otherwise, the volume for the bot will be changed to desired volume if it is within the allowed volume range. | None OR desired volume | No | `!volume 0.5`, `!volume`
 **move** | Moves MumbleDJ into channel if it exists. | Channel | Yes | `!move Music`
+**joinme** | Moves MumbleDJ into your current channel if not playing to someone else| None | Yes | `!joinme`
```
Comment (matthieugrieger) [inline README.md]: Change this line to the following:
```
**joinme** | Moves MumbleDJ into your current channel if not playing audio to someone else. | None | Yes | `!joinme`
```
```

File: commands.go [hunk 2]
```diff
@@ -347,6 +354,16 @@ func move(user *gumble.User, channel string) {
 	}
 }
 
+// move performance !joinme functionality. Found the channel and move the bot. The bot don't move
+// if it already play music to people
```
Comment (matthieugrieger) [inline commands.go]: Change this comment to the following:
```
joinMe performs !joinme functionality. Finds the channel and moves the bot. The bot does not move if it is already playing audio to others.
```
```

File: config.gcfg [hunk 3]
```diff
@@ -106,6 +106,10 @@ HelpAlias = "help"
 # DEFAULT VALUE: "volume"
 VolumeAlias = "volume"
 
+ Alias used for join a user
```
Comment (matthieugrieger) [inline config.gcfg]: Change this line to the following:
```
# Alias used for joinme
```

File: config.gcfg [hunk 4]
```diff
@@ -209,6 +213,10 @@ AdminVolume = false
 # DEFAULT VALUE: true
 AdminMove = true
 
+# Make joinme a admin command?
+# DEFUALT VALUE: true
```
Comment (matthieugrieger) [inline config.gcfg]: Small typo fix: "DEFUALT" should be "DEFAULT".

File: config.gcfg [hunk 5]
```diff
@@ -209,6 +213,10 @@ AdminVolume = false
...
+AdminJoinMe = false
```
Comment (matthieugrieger) [inline config.gcfg]: This should be set to true to match the default value above.

File: strings.go [hunk 6]
```diff
@@ -98,6 +98,9 @@ const SHUFFLE_ACTIVATED_ERROR_MESSAGE = "Automatic shuffle is already activated.
 // Message shown to user when they attempt to disable automatic shuffle while it's already deactivated
 const SHUFFLE_DEACTIVATED_ERROR_MESSAGE = "Automatic shuffle is already deactivated."
 
+// Message show to user when they attempt to move it and it is already playing music to people
```
Comment (matthieugrieger) [inline strings.go]: Change this comment to the following:
```
// Message shown to user when they attempt to move the bot and it is already playing audio to others.
```

File: strings.go [hunk 7]
```diff
@@ -98,6 +98,9 @@ const SHUFFLE_ACTIVATED_ERROR_MESSAGE = "Automatic shuffle is already activated.
...
+const PEOPLE_ARE_LISTENING_TO_ME = "I'm already busy."
```
Comment (matthieugrieger) [inline strings.go]: This message could be a bit more descriptive. How about the following?
```
Users in another channel are listening to me. Please try again later.
```

File: strings.go [hunk 8]
```diff
@@ -151,6 +154,7 @@ const HELP_HTML = `<br/>
 	<p><b>!shuffleon</b> - An admin command that enables auto shuffling.</p>
   	<p><b>!shuffleoff</b> - An admin command that disables auto shuffling.</p>
 	<p><b>!move </b>- Moves MumbleDJ into channel if it exists.</p>
+    	<p><b>!joinme </b>- Move into your channel (MP)</p>
```
Comment (matthieugrieger) [inline strings.go]: The description for this command can just be changed to the one you used above:
```
Moves MumbleDJ into your current channel if not playing audio to someone else.
```

**Comment by matthieugrieger** [comment]
Looks great, thanks for the PR! I added comments for minor issues that should be fixed before the change is merged.

File: strings.go (hunk 7 above)
Comment (azlux) [inline strings.go]: I like short message 'Users in another channel are listening to me.'
no ?

Comment (matthieugrieger) [inline strings.go]: You're right, your message is better. :)

File: config.gcfg [hunk 9]
```diff
@@ -106,7 +106,7 @@ HelpAlias = "help"
 # DEFAULT VALUE: "volume"
 VolumeAlias = "volume"
 
- Alias used for join a user
+ Alias used for joinme
```
Comment (matthieugrieger) [inline config.gcfg]: The `#` that denotes this line as a comment is still missing.

**Comment by matthieugrieger** [comment]
Looks great. Thanks again for the PR!

Merged.

**Comment by azlux** [comment]
I don't know golang and now, i can say I don't really like it, but your code is well done and organized. So it's a pleasure to code into those kind of project.

**Comment by matthieugrieger** [comment]
Haha, thanks! It will be even better once the refactor is done!

**Pull request merged**